edx_proctoring/models.py

```diff
@@ -349,6 +349,10 @@ class ProctoredExamStudentAttempt(TimeStampedModel):
     # else always false
     is_status_acknowledged = models.BooleanField(default=False)
 
+    # if the attempt enters an error state, the remaining time should
+    # be saved in order to allow the learner to resume
+    time_remaining_seconds = models.FloatField(null=True)
```

Comment:
why a float? This seems like a value that would be very well modeled by an integer

Reply:
It was to keep it consistent with the value returned here: https://github.com/edx/edx-proctoring/blob/85a51a3ea05f9555556a1b43bc9a4c7057f36172/edx_proctoring/utils.py#L37

Although I see how it could be confusing. I have no problem changing it.

Reply:
I think keep this as an integer sounds better. Float just complicates code everywhere and we don't need that level of detail.